save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt

```diff
@@ -68,8 +111,8 @@ val createVulnerabilityView: VFC = VFC {
                                         className = ClassName("input-group needs-validation")
                                         textarea {
                                             className = ClassName("form-control")
-                                            onChange = {
-                                                // FIXME: need to add method on backend
+                                            onChange = { event ->
+                                                setVulnerability(vulnerability.copy(description = event.target.value))
```

Comment:
Better do it in functional style: 
```suggestion
                                                setVulnerability { it.copy(description = event.target.value) }
```

Reply:
Fixed